runtime/test_artifacts/generator.go

```diff
@@ -85,35 +85,30 @@ func NewAccounts(n int, privateKeys ...string) (accounts []*coreTypes.Account) {
 //	types of actors which needs to be fixed.
 func NewActors(actorType coreTypes.ActorType, n int) (actors []*coreTypes.Actor, privateKeys []string) {
 	for i := 0; i < n; i++ {
-		genericParam := getServiceUrl(i + 1)
-		if int32(actorType) == int32(coreTypes.ActorType_ACTOR_TYPE_APP) {
-			genericParam = DefaultMaxRelaysString
-		}
-		actor, pk := NewDefaultActor(int32(actorType), genericParam)
+		serviceUrl := getServiceURL(i + 1)
```

Comment:
There's some inconsistency with the `URL` and `Url` suffix here and also I think in the `defaults`, let's pick one and stick to it.